doc/Caliptra.md

```diff
@@ -986,83 +988,82 @@ Each of Caliptra's internal anti-rollback fuse banks shall support a minimum cou
+In this state, since the anti-rollback fuse bank counter is not yet incremented, the Caliptra RTM still allows the firmware to roll back to version 4.2.
 
-Once firmware version 4.3 is fully qualified, the owner will wish to revoke version 4.2. The vendor will issue a follow-up firmware version 4.4, which will place the Calipra RTM in the following state:
+After firmware version 4.3 is fully qualified, the owner wishes to revoke version 4.2. The vendor issues a follow-up firmware version 4.4, which places the Calipra RTM in the following state:
 
 * Currently running firmware version: 4.4
 * Caliptra firmware's signed SVN value: 2
 * Caliptra firmware's signed MIN\_SVN value: 2
 * Caliptra's firmware anti-rollback fuse bank counter value: 2
 
-Upon validating firmware version 4.4, the Caliptra RTM will note that that firmware's signed MIN\_SVN value is 2, and will increment its internal fuse bank counter to match. Once a chip has received firmware version 4.4, it will no longer be able to roll back to version 4.2. However, it *will* continue to be able to roll back to version 4.3.
+Upon validating firmware version 4.4, the Caliptra RTM notes that that firmware's signed MIN\_SVN value is 2, and increments its internal fuse bank counter to match. After a chip receives firmware version 4.4, it is no longer able to roll back to version 4.2. However, it *will* continue to be able to roll back to version 4.3.
 
-## Physical Attack Countermeasures
+## Physical attack countermeasures
 
-A Caliptra RTM shall implement counter measures designed to deter both glitching (also referred to fault-injection (FI) and side-channel attacks (simple power analysis (SPA) and differential power analysis (DPA)).
+A Caliptra RTM shall implement countermeasures designed to deter both glitching (also referred to fault-injection (FI)) and side-channel attacks (simple power analysis (SPA) and differential power analysis (DPA)).
 
 The Caliptra threat model guides the priority of which physical countermeasures are based on a specific physical implementation.
 
-From the top, an adversary in the supply chain has essentially unlimited time to glitch the chip and make it reveal any private key material or symmetric secrets. [One Glitch To Rule Them All ](https://arxiv.org/abs/2108.04575)is one example with recency bias. The most critical counter-measures must prevent non-destructive extraction of those secrets. Otherwise, an adversary succeeding may be able to silently impersonate production serving assets at a later time.
+From the top, an adversary in the supply chain has essentially unlimited time to glitch the chip and make it reveal any private key material or symmetric secrets. [One Glitch To Rule Them All ](https://arxiv.org/abs/2108.04575)is one example with recency bias. The most critical countermeasures must prevent non-destructive extraction of those secrets. Otherwise, an adversary who succeeds can silently impersonate production-serving assets at a later time.
```

Comment:
Space in the wrong place: "\[One Glitch To Rule Them All \](https://arxiv.org/abs/2108.04575)is " -> "\[One Glitch To Rule Them All\](https://arxiv.org/abs/2108.04575) is "

Reply:
fixed